Maths/SumOfDigits.js

```diff
@@ -0,0 +1,44 @@
+/*
+	Gets the sum of the digits of the numbers inputted
+	sumOfDigits(10) will return 1 + 0 = 1
+	sumOfDigits(255) will return 2 + 5 + 5 = 12
+	Wikipedia: https://en.wikipedia.org/wiki/Digit_sum
+*/
+
+/*
+	The given input is converted to a string, split into an array of characters.
+	This array is reversed and joined back together to form a string. which is finally converted to a number.
+*/
+function sumOfDigitsUsingString(number) {
```

Comment:
How is this a valid implementation of the algorithm you propose? This will just reverse the number.

Reply:
Oops, I'm really sorry. I messed up. I'll fix it right away.

Reply:
Fixed the implementation

I intended to write the application I did now, but I kinda messed them up. Extremely sorry for that.